target/linux/mediatek/filogic/base-files/etc/board.d/02_network

```diff
@@ -113,6 +113,9 @@ mediatek_setup_interfaces()
 	mediatek,mt7988a-rfb)
 		ucidef_set_interfaces_lan_wan "lan0 lan1 lan2 lan3 eth2" eth1
 		;;
+	mercusys,mr80x-v3)
+		ucidef_set_interfaces_lan_wan "lan1 lan2 lan3" eth1
+		;;
```

Comment:
This is the same as the existing definition of `abt,asr3000`-`zbtlink,zbt-z8103ax`, please merge with them.